gensrc/thrift/MVMaintenance.thrift

```diff
@@ -34,6 +34,7 @@ enum MVTaskType {
 }
 
 struct TMVMaintenanceStartTask {
+    1: optional Types.TUniqueId query_id;
```

Comment:
Why only `TMVMaintenanceStartTask ` have `query_id`, how can other tasks reuse it?

Reply:
fixed